README.md

```diff
@@ -149,6 +150,7 @@ Node      | Description
 ----------|-------------
 `ganache` | Runs a local [ganache-cli](https://github.com/trufflesuite/ganache-cli) node that is not persistent by default. The contracts from the desired `keeper-contracts` version will be deployed upon launch of this node.
 `spree`   | This is the default. Runs a local node of the Spree Network. See [Spree Network](#spree-network) for details. The contracts from the desired `keeper-contracts` version will be deployed upon launch of this node.
+`duero`   | Runs a local node of the Duero Network and connects to the [Duero Testnet](https://docs.oceanprotocol.com/concepts/testnets/#duero-testnet).
```

Comment:
I will add a subsection about the Duero Testnet to the docs today. This link should work once that is done.

Reply:
See https://github.com/oceanprotocol/docs/pull/250